packages/@stylexjs/inline-css/src/index.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+const valueProxy = (_propName) =>
+  new Proxy(function () {}, {
+    get() {
+      return valueProxy('');
+    },
+    apply() {
+      return undefined;
+    },
+  });
```

Comment:
It should probably not be a recursive proxy and the `get()` should throw an error similar to the stylex package saying you need to compile this away.